### PR TITLE
Improvements to the English version of the "what" page

### PR DESCRIPTION
**Pull request by axelsimon**
Hi!
Here are two commits to help improve the "what" page:
- one for improving the language of the page
- one to try and fix the size of the Peertube logo, so it aligns with the Youtube one

Hope this helps!

### CODE REVIEW

File: content/page/what.en.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ---
 title: "What"
-date: 2018-01-11 17:33:02
+date: 2020-07-23 17:45:00
 draft: false
 ---
 
-# What Exodus Privacy does
+# What εxodus Privacy does
```
Comment (pnu-s) [inline content/page/what.en.md:7]: I know it's not always clear, but Exodus Privacy is the organization, εxodus the tool.
So there is no such thing as εxodus Privacy ;)

File: content/page/what.en.md [hunk 2]
```diff
@@ -73,33 +73,33 @@ If you are curious or want to help us to improve εxodus, you can file issues or
...
 
 <hr>
 
 <a name="videos"></a>
 
-## Popularization supports
+## Outreach: Learn More
 
-Exodus Privacy main interest is to make people aware of the tracking done by mobile applications. To do so, we are creating videos explaining how and why tracking massively occurs on smartphones. These videos are available on 2 platforms:
+The main goal of εxodus Privacy is to raise awareness of the extent of tracking by and in mobile applications. To this end, we have created videos explaining how and why tracking massively occurs on smartphones. You can watch these videos on 2 platforms:
```
Comment (pnu-s) [inline content/page/what.en.md:99]: Same here, Exodus Privacy and not εxodus Privacy

File: content/page/what.en.md [hunk 3]
```diff
@@ -73,33 +73,33 @@ If you are curious or want to help us to improve εxodus, you can file issues or
...
 <div class="row justify-content-md-center">
   <div class="col-md-4 text-center">
-    <img src="/media/peertube.svg" height="48px" class="mt-3 ml-auto mr-auto"/>
+    <img src="/media/peertube.svg" height="42px" class="mt-3 ml-auto mr-auto"/>
```
Comment (pnu-s) [inline content/page/what.en.md:102]: Could you please make the same change on the french version of this page? :)

Comment (axelsimon) [reply content/page/what.en.md:102]: Hi! Yes, but that will be a different PR :)

File: content/page/what.en.md (hunk 1 above)
Comment (axelsimon) [reply content/page/what.en.md:7]: Thanks, understood, i'll make the change.

File: content/page/what.en.md (hunk 3 above)
Comment (axelsimon) [reply content/page/what.en.md:102]: Sure.

File: content/page/what.en.md (hunk 2 above)
Comment (axelsimon) [reply content/page/what.en.md]: Got it! Changed.

File: content/page/what.en.md (hunk 3 above)
Comment (axelsimon) [reply content/page/what.en.md:102]: Added a commit for this change :)

**Comment by axelsimon** [comment]
Change requests have been addressed (and i learned something about git in the process :tada:), ready for re-review!

**Comment by pnu-s** [comment]
Awesome, thanks a lot @axelsimon :+1:

**Pull request merged**